src/libraries/Common/src/Interop/Windows/IpHlpApi/Interop.ICMP.cs

```diff
@@ -54,13 +54,21 @@ internal struct IcmpEchoReply
         }
 
         [StructLayout(LayoutKind.Sequential, Pack = 1)]
-        internal struct Ipv6Address
+        internal unsafe struct Ipv6Address
         {
-            [MarshalAs(UnmanagedType.ByValArray, SizeConst = 6)]
-            internal byte[] Goo;
-            [MarshalAs(UnmanagedType.ByValArray, SizeConst = 16)]
+            internal fixed byte Goo[6];
+
             // Replying address.
-            internal byte[] Address;
+            private fixed byte _Address[16];
+            internal byte[] Address
+            {
+                get
+                {
+                    fixed (byte* pAddress = _Address)
+                        return new ReadOnlySpan<byte>(pAddress, 16).ToArray();
+                }
+            }
```

Comment:
```suggestion
            internal byte[] Address => MemoryMarshal.CreateReadOnlySpan(ref _Address[0], 16).ToArray();            
```
?

Reply:
Also fixed the same pattern from previous PR